inference-engine/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -26,6 +26,11 @@
 namespace ov {
 namespace test {
 
+std::ostream& operator <<(std::ostream& os, const InputShape& inputShape) {
+    os << inputShape.first << "_" << CommonTestUtils::vec2str(inputShape.second);
```

Comment:
Standard partial shape dump uses comma separators, could you please use `CommonTestUtils::partialShape2str` here?